app/jobs/regular/pull_hotlinked_images.rb

```diff
@@ -9,6 +9,89 @@ def initialize
       @max_size = SiteSetting.max_image_size_kb.kilobytes
     end
 
+    def execute(args)
+      @post_id = args[:post_id]
+      raise Discourse::InvalidParameters.new(:post_id) unless @post_id.present?
```

Comment:
```suggestion
      raise Discourse::InvalidParameters.new(:post_id) if @post_id.blank?
```
This is trivial but usually it's easier for our brains to parse regular ifs rather than unless/not.